src/components/FileUploader/FileUploader.tsx

```diff
@@ -10,17 +11,23 @@ interface FileUploaderProps {
 }
 
 export const FileUploader: FC<FileUploaderProps> = ({uploadLink, acceptedFormats, refetch}) => {
+  const {mutate: fileUpload} = useMutation({
+    mutationFn: (formData: FormData) => {
+      return axios.post(uploadLink, formData)
+    },
```

Comment:
aj tu by som to skratil:
```suggestion
    mutationFn: (formData: FormData) => axios.post(uploadLink, formData),
```